docs/content/en/docs/concepts/architecture/components/scheduler/_index.md

```diff
@@ -14,3 +14,152 @@ The Lifecycle Scheduler can trigger events such as
 deployment, testing, and remediation at specific times or intervals.
 The Keptn Scheduler guarantees that Pods are initiated only after
 the Pre-Deployment checks are completed.
+
+## Architecture of Keptn Scheduler
+
+To integrate the `Keptn Lifecycle Scheduler` into a Kubernetes cluster,
+a Kubernetes manifest is applied with specific **Keptn annotations**.
+These annotations identify the workload and specify the
+pre-deployment and post-deployment tasks and evaluations
+associated with it.
+
+## How it works
+
+The `Keptn Scheduler`, integrated into the `Keptn Lifecycle Operator`,
+which is equipped with multiple controllers for managing
+**Keptn Custom Resource Definitions (CRDs)** and a powerful
+**Mutating Webhook**.
+Its core responsibility is to orchestrate the scheduling of Pods, taking
+into account pre-deployment checks to ensure optimal deployment success.
+
+Let's dive into its architectural details:
+
+- **Scheduler Flow**:
+After applying a Kubernetes Manifest annotated with Keptn-specific
+annotations to the cluster, the Mutating Webhook injects the Keptn Scheduler.
+The Scheduler then receives Kubernetes Events for
+Pre-Deployment, which are sent to the event stream.
```

Comment:
Be careful here...  With just the basic annotations, you get the observability features but do not get the pre- and post-deployment evaluations and tasks -- those are separate.  The existing documentation is a bit misleading here -- PR coming soon that attempts to correct it.  Apparently the annotations/labels for the evaluations and tasks are applied to the Kubernetes Workloads and affect only the KeptnWorkloads individually.  The pre- and post-deployment evaluations and tasks are added manually to the KeptnApp resource by name,   People may run the observability features without using the release lifecycle management features (which are the tasks/evaluations applied to the entire KeptnApp so we need to describe here how the Scheduler manages each different piece.